internal/batches/ui/json_lines.go

```diff
@@ -149,7 +149,7 @@ func (ui *JSONLines) CreatingBatchSpecSuccess(batchSpecURL string) {
 	})
 }
 
-func (ui *JSONLines) CreatingBatchSpecError(lr int, err error) error {
+func (ui *JSONLines) CreatingBatchSpecError(maxUnlicensedCS int, err error) error {
```

Comment:
```suggestion
func (ui *JSONLines) CreatingBatchSpecError(_ int, err error) error {
```

Since aren't making use of this variable and we are only passing it here to satisfy the interface we can safely ignore it using the underscore. That way the linter doesn't complain about this.